internal/providers/docker/compose/command.go

```diff
@@ -1,7 +1,14 @@
 package compose
 
-import "mvdan.cc/sh/v3/shell"
+import (
+	"github.com/mattn/go-shellwords"
+)
 
+// FIXME: this incorrectly treats the "command" passed to docker as a
+// meaningful shell command that can be parsed into "shell words". In reality
+// this is an arbitrary string that is passed to the entrypoint of the
+// container as an argument. It could be a shell command but it could also be
+// a python program or a Dan Brown novel.
```

Comment:
Rather than a joke here, would have been nice to explain the problem with reference to the spec and or what to do about it. I've re-opened #217 and will add context there.